docs/user-guide/administration/manage-etcd.md

```diff
@@ -245,3 +300,178 @@ spec:
 ```
 
 Restart GreptimeDB Metasrv to complete etcd restore.
+
+## Monitoring
+
+- Prometheus Operator installed (e.g. via [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)).
+- podmonitor CRD installed (automatically installed with Prometheus Operator).
+
+Add the following to your `etcd-monitoring.yaml` to enable monitoring:
+
+```yaml
+replicaCount: 3
+
+auth:
+  rbac:
+    create: false
+  token:
+    enabled: false
+
+persistence:
+  storageClass: null
+  size: 8Gi
+
+resources:
+  limits:
+    cpu: '1'
+    memory: 2Gi
+  requests:
+    cpu: '1'
+    memory: 2Gi
+
+autoCompactionMode: "periodic"
+autoCompactionRetention: "1h"
+
+extraEnvVars:
+  - name: ETCD_QUOTA_BACKEND_BYTES
+    value: "8589934592"
+  - name: ETCD_ELECTION_TIMEOUT
```

Comment:
Based on issue https://github.com/etcd-io/etcd/issues/13889, we also need to set up `--snapshot-count`. 

If the user uses <3.6, it's better to use `10000`.

Reply:
Fixed